core-bundle/src/Util/ProcessUtil.php

```diff
@@ -57,7 +57,13 @@ public function createSymfonyConsoleProcess(string $command, string ...$commandA
     {
         // Use PhpSubprocess introduced in Symfony 6.4 to respect command line arguments
         // used to invoke the current process.
-        return new PhpSubprocess([$this->getPhpBinary(), $this->getConsolePath(), $command, ...$commandArguments]);
+        return new PhpSubprocess(
+            [$this->getConsolePath(), $command, ...$commandArguments],
+            null,
+            null,
+            60,
+            [$this->getPhpBinary()],
+        );
```

Comment:
```suggestion
        return new PhpSubprocess([$this->getConsolePath(), $command, ...$commandArguments], php: [$this->getPhpBinary()]);
```

Feel free to ignore if you want to provide the default values explicitly.

Reply:
Are we using named arguments anywhere in Contao?

Reply:
Yes, e.g. here:

https://github.com/contao/contao/blob/1edc301296f43f76b01989e10ad77fcab3d3de2c/core-bundle/contao/library/Contao/Environment.php#L412